icssc/AntAlmanac
# Show delete button on course search page

## Summary
When you add a class from the course search page, the add button is replaced by the color and delete buttons.
![antalmanac deletebutton](https://user-images.githubusercontent.com/48658337/155907151-15b4b42a-bab5-4f1a-8b02-ceef75ec6c0d.gif)
(gif generated with [ScreenToGif](https://www.screentogif.com/); I highly recommend it)
## Test Plan
Actions tested:
- add a class
- delete class from course search
- delete class from calendar
- delete class from added class tab
- change color from course search
- change color from added class tab
- change color from calendar (this doesn't update the color picker immediately in either course search or added classes, but this was a bug beforehand. I'm gonna create a separate issue for that)
- switch tabs, confirm that class stays added or deleted
## Issues
Closes #237 
## Future Followup (Optional)
Identified a new bug, going to create a new issue for that. (#263)
